rpcs3/Emu/RSX/GL/GLPresent.cpp

```diff
@@ -149,7 +149,8 @@ void GLGSRender::flip(const rsx::display_flip_info_t& info)
 		if (!buffer_pitch)
 			buffer_pitch = buffer_width * avconfig.get_bpp();
 
-		const u32 video_frame_height = (!avconfig._3d ? avconfig.resolution_y : (avconfig.resolution_y - 30) / 2);
+		//gonna come back to this once i settle on how to handle this with modes, it makes sense for anaglyph but not SBS or over-under
+		const u32 video_frame_height = (avconfig.stereo_mode == stereo_render_mode_options::anaglyph? (avconfig.resolution_y - 30) / 2 : avconfig.resolution_y);
```

Comment:
I think he means this
```suggestion
		const u32 video_frame_height = (avconfig.stereo_mode == stereo_render_mode_options::disabled ? avconfig.resolution_y : (avconfig.resolution_y - 30) / 2);
```

Reply:
yeah i knew about that, just wasn't sure about the general other logic, maybe it'll become more acceptable once i get the modes working with the proper buffer size.

thanks for the clarification, i'll work on that, although it'll be a bit because i'm not home and VNC over a slow connection isn't ideal.